### ISSUE TITLE
1.3.1 - Call to undefined function array_column()

### ISSUE THREAD

**Issue by mathieuchateau** [pluginsGLPI/fields#76]
After upgrading to 1.3.1, trying to create a new block:

```
Fatal error: Call to undefined function array_column() in /glpi/www/glpi/plugins/fields/inc/container.class.php on line 261
```

**Comment by mathieuchateau** [comment]
only appears if trying to create a block with same name as existing one but on different scope (computers vs users).
OK if create with a new name, then rename to conflicting name, as internal name are still different

**Comment by trasher** [comment]
This function is available on PHP 5.5 and greater. On GLPi side, a lib (`ramsey/array_column`) has been used to make that working on older versions.

**Comment by trasher** [comment]
In facts, the plugin in its last version is actually compatible with GLPi 9.1 only.

**Comment by orthagh** [comment]
or php 5.5, no ?

**Comment by trasher** [comment]
Yep, indeed :)

**Comment by orthagh** [comment]
i suggest so to do nothing.
With time, we'll have more and more recent php or glpi  versions.
Or maybe just a warning when none of this prerequisites fullfiled.

**Comment by trasher** [comment]
+1 for a check of the presence of the required function; thanks @orthagh

**Comment by trasher** [comment]
Will be fixed in the 1.3.2 release.
